anvil/src/render.rs

```diff
@@ -202,7 +203,13 @@ pub fn render_output<'a, R>(
     damage_tracker: &mut OutputDamageTracker,
     age: usize,
     show_window_preview: bool,
-) -> Result<(Option<Vec<Rectangle<i32, Physical>>>, RenderElementStates), OutputDamageTrackerError<R>>
+) -> Result<
+    (
+        Option<(SyncPoint, Vec<Rectangle<i32, Physical>>)>,
+        RenderElementStates,
+    ),
+    OutputDamageTrackerError<R>,
+>
```

Comment:
This type is getting a bit unwieldy (I am actually a little surprised clippy is not warning on this tbh). Maybe we can put the `Result::Ok` part in a new struct?

Reply:
It does warn, but it was hidden behind the `glow` build error ^^ I already changed it to use a `OutputRenderResult`